use_cases/eluc/.gitignore

```diff
@@ -0,0 +1,4 @@
+predictors/*/*
```

Comment:
This is to prevent pushing the saved models

Reply:
Whoops, yes. This ignores all the predictor implementations. I'll use another PR for those but for now there are tests that correspond to models that don't exist yet.